templates/semantic-ui/zenodo_rdm/records/detail.html

```diff
@@ -64,6 +64,33 @@ <h4 class="inline">{{ record.ui.access_status.title_l10n }}</h4>
 
 {# Additional details #}
 {%- block record_details -%}
+
+  {% if record.expanded and record.expanded.parent.communities.default.slug == "biosyslit" %}
```

Comment:
minor: it would be nice if there was a way to check this very nested attribute access (e.g. perhaps JS optional chaining) to avoid Jinja blowing up the page with an exception.

I would also extract this into a variable at the top of the template (to re-use below in the `javascript` block)